sdk/src/dlob/orderBookLevels.ts

```diff
@@ -139,12 +140,20 @@ export function getVammL2Generator({
 	oraclePriceData,
 	numOrders,
 	now,
+	topofBookQuoteAmounts,
```

Comment:
why'd you make it in quote instead of base?

Reply:
so its easy to make consistent across perp markets, thus comparing spreads across perp markets would be more consistent